### DISCUSSION TITLE
how to select = in Fundamental mode

### DISCUSSION THREAD

**Discussion by capfredf** [meow-edit/meow#699] [General]
say I have text like this:
```
aaa = b + c
ddddddd = m + d
```
I want to change it to
```
aaa = 42 + b + c
ddddddd = 42+ m + d
```
My approach is to select = in `beacon-mode` by pressing `W` and press `a` to insert `42 + `. However, meow does not allow to select symbols in this case. FWIW, this workflow is very similar to what I used to when I was using multiple-cursor.  Is there a way to select anything under the cursor in Meow? Also, what are recommended ways to I should try instead?

**Comment by formsandlines** [comment]
`meow-mark-symbol` should work on symbols like `=` and `+`, I just tested it. I also use `meow-find` + the character you want to select for things like this. I am not sure what the default bindings were though since I heavily customized my own. :)

**Reply by capfredf** [reply]
OOTB, It depends on the major mode being used. For example. it works with the elisp mode and racket-mode, but not with the haskell mode or the fundamental mode.

**Comment by DogLooksGood** [comment]
In this case, you can use `meow-find` in beacon state. Or you can use  char movement or `isearch` to select one `=` character, then do `meow-search`.
